telegram/ext/jobqueue.py

```diff
@@ -485,6 +485,14 @@ def next_t(self):
 
     @next_t.setter
     def next_t(self, next_t):
+        if isinstance(next_t, datetime.datetime):
+            # Set timezone to UTC in case datetime is in local timezone.
+            next_t = next_t.replace(tzinfo=datetime.timezone.utc)
```

Comment:
I think we'd rather want `next_t.astimezone(UTC)` here, looking at the [docs](https://docs.python.org/3/library/datetime.html#datetime.datetime.astimezone). Do you agree?

Reply:
Sure, it will be more readable.